doc/source/quickstart.rst

```diff
@@ -122,7 +122,7 @@ the SDK packages needed.
 For Android NDK, note that modern releases will only work on a 64-bit
 operating system. **The minimal, and recommended, NDK version to use is r19b:**
 
- - `Go to ndk downloads page <https://developer.android.com/ndk/downloads/>`_
+ - `Go to ndk downloads page <https://github.com/android/ndk/wiki/Unsupported-Downloads#r19c>`_
```

Comment:
Thanks for the pull request.
I think we also have one more reference above from the same documentation

Reply:
That's just about that you should use any version but r19c is maximum